sqldelight-gradle/plugin/src/main/kotlin/com/squareup/sqldelight/gradle/VerifyMigrationTask.kt

```diff
@@ -36,8 +38,14 @@ open class VerifyMigrationTask : SourceTask() {
   }
 
   @TaskAction
-  fun generateSchemaFile() {
-    val currentDb = DriverManager.getConnection("jdbc:sqlite:")
+  fun verifyMigrations() {
+    var currentDb: Connection
+    try {
+      currentDb = DriverManager.getConnection("jdbc:sqlite:")
+    } catch (e: SQLException) {
+      currentDb = DriverManager.getConnection("jdbc:sqlite:")
```

Comment:
JDBC is the fucking worst. How about
```kotlin
val currentDb = try { .. } catch (..) { .. }
```
though?

Reply:
it truly is. works for me.